src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+  font-style: normal;
+}
+
+.page {
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.page__search-bars {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  margin: 20px 8px 0;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 0 0 62px;
+
+  font-family: inherit;
+  font-weight: inherit;
+  font-size: 16px;
+  color: inherit;
+
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 50%;
+
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
```

Comment:
Great job!

One more thing - input should have a gradient on focus.
Taking into account you already have the background image you should combine 2 backgrounds: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_backgrounds_and_borders/Using_multiple_backgrounds), [W3S](https://www.w3schools.com/css/css3_backgrounds.asp)